dts/x86/intel/raptor_lake.dtsi

```diff
@@ -44,6 +45,25 @@
 		#address-cells = <1>;
 	};
 
+	acpi0: acpi0 {
```

Comment:
ACPI is not a hw bus. This is going against DTS idea.

I think we already discussed that.

Reply:
removed ACPI node. In order to distinguish whether a driver use ACPI interface or not, one can use DT_NODE_HAS_PROP(n, acpi_uid)). 

@galak could you please have a look and share your though ?